main.rb

```diff
@@ -1,46 +1,54 @@
+require './refactor'
 require './app'
 
-def menu(choice)
-  case choice
-  when 1
-    display_books
-  when 2
-    list_of_people
-  when 3
-    create_person
-  when 4
-    create_new_book
-  when 5
-    create_rental
-  when 6
-    rent_list_by_id
+class Main
+  def initialize
+    @app = App.new
+    @refactor = Refactor.new
   end
-end
+  # def start
+  #   puts 'Welcome to our School Library App'
+  #   app = Refactor.new
+  #   app.user_input
+  # end
 
-def user_input
-  puts "
-  Welcome to our Library! You may perform the following tasks from the Menu:\n
-        1 - List all books
-        2 - List all people
-        3 - Create a person
-        4 - Create a book
-        5 - Create a rental
-        6 - List all rentals for a given person (id)
-        7 - Exit
-        Choose(Any option from 1-7): "
-  input_value = gets.chomp.to_i
-  menu(input_value)
-  if input_value == 7
-    puts '********** Thank you for visiting our library  ********'
-  else
-    user_input
+  def user_input
+    puts "
+    Welcome to our Library! You may perform the following tasks from the Menu:\n
+          1 - List all books
+          2 - List all people
```

Comment:
- [ ] The list all people method in your `app.rb` is no longer working as expected. When I try to list all people, a message is displayed as follows: "No People Registered"
![image](https://user-images.githubusercontent.com/46839548/195274503-68d379d8-11b8-47d7-a18e-1ab0348ca152.png)
Please try to fix it 👍